src/test/java/uk/gov/hmcts/reform/pip/data/management/service/schemavalidation/MagistratesPublicListTest.java

```diff
@@ -0,0 +1,442 @@
+package uk.gov.hmcts.reform.pip.data.management.service.schemavalidation;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.zonky.test.db.AutoConfigureEmbeddedDatabase;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.pip.data.management.Application;
+import uk.gov.hmcts.reform.pip.data.management.config.AzureBlobConfigurationTest;
+import uk.gov.hmcts.reform.pip.data.management.errorhandling.exceptions.PayloadValidationException;
+import uk.gov.hmcts.reform.pip.data.management.models.publication.ListType;
+import uk.gov.hmcts.reform.pip.data.management.service.ValidationService;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@SpringBootTest(classes = {Application.class, AzureBlobConfigurationTest.class})
+@ActiveProfiles(profiles = "test")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
+@AutoConfigureEmbeddedDatabase(type = AutoConfigureEmbeddedDatabase.DatabaseType.POSTGRES)
+
+class MagistratesPublicListTest {
+    @Autowired
+    ValidationService validationService;
+
+    private static final String MAGISTRATES_PUBLIC_LIST_VALID_JSON =
+        "mocks/magistrates_public_list/magistratesPublicList.json";
+    private static final String MAGISTRATES_PUBLIC_LIST_INVALID_MESSAGE =
+        "Invalid crime magistrates public list marked as valid";
+
+    private static final String COURT_LIST_SCHEMA = "courtLists";
+    private static final String VENUE_SCHEMA = "venue";
+    private static final String VENUE_ADDRESS_SCHEMA = "venueAddress";
+    private static final String VENUE_CONTACT_SCHEMA = "venueContact";
+    private static final String COURT_HOUSE_SCHEMA = "courtHouse";
+    private static final String COURT_ROOM_SCHEMA = "courtRoom";
+    private static final String SESSION_SCHEMA = "session";
+    private static final String SITTINGS_SCHEMA = "sittings";
+    private static final String HEARING_SCHEMA = "hearing";
+    private static final String CASE_SCHEMA  = "case";
+
+    private JsonNode getJsonNode(String json) throws JsonProcessingException {
+        ObjectMapper mapper = new ObjectMapper();
+        return mapper.readValue(json, JsonNode.class);
+    }
+
+    @Test
+    void testValidateWithErrorsWhenDocumentMissingInFamilyDailyCauseList() throws IOException {
```

Comment:
Junaid, just notice this. You need to change the name for all the test method in this class. Too much copy and paste!! :)

Reply:
fixed :)